xbmc/addons/kodi-dev-kit/include/kodi/versions.h

```diff
@@ -108,7 +108,7 @@
 #define ADDON_INSTANCE_VERSION_IMAGEDECODER_DEPENDS   "c-api/addon-instance/imagedecoder.h" \
                                                       "addon-instance/ImageDecoder.h"
 
-#define ADDON_INSTANCE_VERSION_INPUTSTREAM            "3.1.0"
+#define ADDON_INSTANCE_VERSION_INPUTSTREAM            "3.1.1"
```

Comment:
Not sure ADDON_INSTANCE_VERSION_INPUTSTREAM_MIN   must also be bumped, as a new enum value was inserted not at the end of the enum.

Reply:
Good point, I would just stick the enum value at the end so we don't have to bump minor.

Reply:
Yeah but there is a last enum value with a special cb semantics at the end of this enum if I see this right, so we can’t spend new values.